app/src/main/java/com/mobile/tandil/kotlinbaseproject/fragment/MainDialogFragment.kt

```diff
@@ -0,0 +1,52 @@
+package com.mobile.tandil.kotlinbaseproject.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import com.mobile.tandil.kotlinbaseproject.databinding.FragmentMainDialogBinding
+import com.mobile.tandil.kotlinbaseproject.listener.ListenerSetParkingSpaces
+import com.mobile.tandil.kotlinbaseproject.mvvm.viewmodel.MainDialogViewModel
+import com.mobile.tandil.kotlinbaseproject.utils.Constants
+
+class MainDialogFragment() : DialogFragment() {
```

Comment:
pls use a name more representative, to indicate in code whats the propose of this Dialog. Suggestion: ParkingSizeDialogFragmet.
Also, do the same to the ViewModel